apiary.apib

```diff
@@ -518,6 +518,28 @@ Get a list of all the definitions of custom fields.
 
     + Attributes (object)
         + page (Page, optional)
+        + filter (object, optional)
+            + context (enum[string], optional)
+                + Members
+                    + contact
+                    + company
+                    + form
+                    + event
+                    + product
+                    + project
+                    + milestone
+                    + sale
+                    + project_thirdparty
```

Comment:
B : this is not an existing type as far as the API is concerned
we are deprecating third parties in this form